config/v1/types_infrastructure.go

```diff
@@ -170,6 +174,24 @@ type BareMetalPlatformStatus struct {
 	NodeDNSIP string `json:"nodeDNSIP,omitempty"`
 }
 
+// OpenStackPlatformStatus holds the current status of the OpenStack infrastructure provider.
+type OpenStackPlatformStatus struct {
```

Comment:
Need the `CloudName` for OpenStack's platform status. Otherwise the registry operator will not know which item in `clouds.yaml` it should use to configure swift storage.

See https://github.com/openshift/installer/issues/1986

Reply:
I've added the `CloudName` field.